versioned_docs/version-3.x/reference/zmodel/data-field.md

```
@@ -0,0 +1,77 @@
+type Type {
+    FIELD_NAME FIELD_TYPE (FIELD_ATTRIBUTES)?
+}
+```
+
+-   **FIELD_NAME**
+
+    Name of the field. Needs to be unique in the containing model or type. Must be a valid identifier.
+
+-   **FIELD_TYPE**
+
+    Type of the field. Can be a scalar type, a reference to another model or type if the field belongs to a [model](#model), or a reference to another type if it belongs to a [type](#type).
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)